theme/js/components/discussions/thread.vue

```diff
@@ -80,62 +74,59 @@
 import ThreadReply from "./thread-reply.vue";
 import Avatar from "./avatar.vue";
 import Author from "./author.vue";
-import LinkIcon from "svg/permalink.svg";
 import config from "../../config";
 
 export default {
+  components: {
+    "thread-reply": ThreadReply,
+    Avatar,
+    Author,
+  },
+  props: {
+    id: String,
+    discussion: Array,
+    title: String,
+    url: String,
+    closed: String,
+    closed_by: Object,
+  },
   data() {
     return {
-      showForm: false, //User has clicked on the "add comment" btn
-      updatedDiscussion: null, //This is a bit ugly, we hold two states here for the updated discussion when the user replies. Should probably hoist this up.
-      LinkIcon, //Little svg loading hack
-      collapsed: true, //The thread is collapsed by default for closed discussions
+      showForm: false,
+      updatedDiscussion: null,
+      collapsed: true,
       readOnlyEnabled: config.read_only_enabled,
     };
   },
   computed: {
-    _discussion: function () {
-      //And this is the logic to get either the original discussion passed in prop or the updated one
+    _discussion() {
+      // Discussion updates are saved locally only
+      // This is the logic to get either the original discussion or the updated one
       return this.updatedDiscussion ? this.updatedDiscussion : this.discussion;
     },
     _collapsed() {
       return this.closed && this.collapsed;
     },
   },
-  components: {
-    "thread-reply": ThreadReply,
-    Avatar,
-    Author,
-  },
   methods: {
-    discussionUrl: (id, link = false) => (link ? "#" : "") + "discussion-" + id, //Permalink helpers
-    commentUrl: function (id, index, link = false) {
```

Comment:
When talking about the missing elements from Figma, we decided to remove it because there is no visual difference between a discussion permalink and a comment permalink :
- the message is the same
- the page scroll at the top of the discussion
- there is no visual information about the comment linked

But we can kept them as is or update them.